temboardagent/plugins/monitoring/__init__.py

```diff
@@ -164,6 +168,51 @@ def api_run_probe(probe_instance, config):
     return run_probes([probe_instance], [instance], delta=False)
 
 
+@routes.get(b'/', check_key=True)
+def get_monitoring(http_context, app):
+    """Monitoring root API aims to query metrics history.
+    Data are sorted by collect timestamp, in ascending order. By default, only
+    the most fresh record set is returned. The query parameter 'start' can be
+    used as lower bound and must be expressed as a UTC timestamp formatted
+    using ISO8601 norm with a terminal 'Z' character. To limit the number of
+    returned records to N, the query parameter 'limit' can be used and set to
+    N. 'limit' default value is 50, meaning that the maximum number of record
+    set this API returns by default is 50.
+    """
+
+    # Default values
+    start_timestamp = None
+    limit = 50
+
+    if 'start' in http_context['query']:
+        # Validate start parameter
+        validate_parameters(http_context['query'], [
+            ('start', T_TIMESTAMP_UTC, True),
+        ])
```

Comment:
This looks a bit fragile; for instance "1234-42-43T87:99:88Z" would match this regex but fail to be parsed by `datetime.strptime()` below. So perhaps, we could just pass the value to strptime(), catch possible `ValueError` and raise `HTTPError` is that happens? Or maybe have `validate_parameters()` handle timestamps directly?

Reply:
Done.